rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize
+    @date = date
+    @person = person
+    @book = book
+    person.rentals << self
+    book.rentals << self
```

Comment:
Nice job for achieving this belongs to relationship :100: (A rental itself belongs to a @book  and a @person:+1: )